docs/source/topics/proc_installing-codeready-containers.adoc

```diff
@@ -1,11 +1,57 @@
 [id="installing-codeready-containers_{context}"]
 = Installing {prod}
 
+{prod} is available as a portable executable for {rhel} and {msw}.
+On {mac}, {prod} is available using a guided installer.
+
 .Prerequisites
 
 * Your host machine must meet the minimum system requirements.
 For more information, see link:{crc-gsg-url}#minimum-system-requirements_gsg[Minimum system requirements].
 
 .Procedure
 
-. Download the link:{crc-download-url}[latest release of {prod}] for your platform and extract the contents of the archive to a location in your `_PATH_`.
+. Download the link:{crc-download-url}[latest release of {prod}] for your platform.
+
+. On {mac}, run the guided installer and follow the instructions.
+On {rhel} or {msw}, assuming the archive is in the [filename]*_~/Downloads_* directory, follow the procedure for your platform.
++
+** For {rhel}:
++
+... Extract the contents of the archive:
++
+[subs="attributes"]
+----
+$ cd ~/Downloads
+$ tar xvf crc-linux-amd64.tar.xz
+----
++
+... Create the [filename]*_~/bin_* directory if it does not exist and copy the [command]`{bin}` executable to it:
++
+[subs="attributes"]
+----
+$ mkdir -p ~/bin
+$ cp ~/Downloads/crc-linux-*-amd64/{bin} ~/bin
+----
++
+... Add the [filename]*_~/bin_* directory to your `_PATH_`:
++
+[subs="attributes"]
+----
+$ export PATH=$PATH:$HOME/bin
+$ echo 'export PATH=$PATH:$HOME/bin' >> ~/.bashrc
+----
++
+** For {msw}:
++
+... Extract the contents of the archive.
+... Create the desired directory path for {prod} and copy the extracted [command]`{bin}.exe` executable to it.
+... Add the directory path to your `_PATH_` environment variable.
++
+[NOTE]
+====
+On {msw}, you must execute the {prod} executable from your local [filename]*_C:\_* drive.
+You cannot run {prod} from a network drive.
+
+{prod} can be run from the current directory as [command]`./{bin}.exe` if you cannot install the executable in your `_PATH_`.
```

Comment:
So I don't think the IBM SG specifically dictates that subordinate clauses must begin a sentence instead of end a sentence. However, most of the SG's examples of `if` clauses are at the beginning of the sentence, not the end. So consider rephrasing to:

`If you cannot install the executable in your `_PATH_`, {prod} can be run...`

Also, for consistency, phrase the second sentence in active voice, directly talking to the user, like you do in the first sentence. So consider rephrasing to something like, 

`If you cannot install the executable in your `_PATH_`, run {prod} from your current directory as [command] `./{bin}.exec`. `

Reply:
Good point, fixing that now. Thanks!